### DISCUSSION TITLE
MPC Hacking

### DISCUSSION THREAD

**Discussion by jdefrancesco** [TheKikGen/USBMidiKliK4x4#44] [General]
Is this the MPC hacking group? I recently rooted my MPC One and have been reversing the binary with BinaryNinja. Maybe we can start a forum or something so we can all share our findings? Dunno if anyone is interested but I have a pretty good overview of their system..

**Comment by TheKikGen** [comment]
No.  This is the group for USB MidiKlik 4x4.  You should open a discussion on the MPCLiveXPlore project.
I also use Binary Ninja. Happy to exchange.  https://github.com/TheKikGen/MPC-LiveXplore/discussions
